server/definitions/IBinder.ts

```diff
@@ -5,7 +5,7 @@ export const binderAlgorithmName = "de.metanome.algorithms.binder.BINDERFile";
 export interface IBinderConfig extends IMetanomeConfig {
   // whether to detect nary inds at all
   // should be set automatically
-  DETECT_NARY?: boolean; //=false
+  // DETECT_NARY?: boolean; //=false
```

Comment:
Kann vielleicht dann ganz gelöscht werden?

Reply:
Geht defitnitiv auch. Ich würd's dalassen, damit ist "dokumentiert" welche Optionen Binder in Metanome benutzt.
Habe den Kommentar ein wenig angepasst. Ich kann es aber auch gerne rausnehmen :)